### ISSUE TITLE
Import for Value as Concept in Measurement criterion not working

### ISSUE THREAD

**Issue by cgreich** [OHDSI/Atlas#2160]
Opens a box to paste concept ids in, but the "Done" button is not working.

**Comment by Alexdavv** [comment]
It's all the same among all the criteria types and attributes - Import function is not working.

**Comment by anthonysena** [comment]
Presumably this problem arose during the COVID19 work so noting that here since it is a problem on v2.7.6. That said, I was unable to reproduce this problem when using valid concept IDs separated by a comma. 

@cgreich @Alexdavv: if you happen to have the concept IDs you used and the format (i.e. space delimited, comma delimited, etc) this would help to focus the troubleshooting. The only way I was able to get this to fail was to provide an invalid concept_id in the list. For example, I put in "-1" and the failure happens when attempting to resolve that concept_id against the vocabulary source - it fails since the concept id is invalid.

**Comment by Alexdavv** [comment]
> if you happen to have the concept IDs you used and the format (i.e. space delimited, comma delimited, etc) this would help to focus the troubleshooting.

We used the list delimited by paragraph symbol - it works in Search - Import, but not here. If commas added, it works then.

The original list:
4126681
45877985
9191
45884084
4181412
45879438

**Comment by chrisknoll** [comment]
Ok, so we'll add a fix to split the input based on comma or whitespace characters.

**Comment by chrisknoll** [comment]
Ok, I checked and this should be fixed in 2.8: this is the code that we're using to pull out numbers from the concept ID import textbox:
```
vocabularyApi.getConceptsById(this.identifiers().match(/[0-9]+/g));
```

This will match any sequential number, and create the concept ID list from the set of matches.
